scripts/lint-style.py

```diff
@@ -203,6 +203,8 @@ def four_spaces_in_second_line(lines, path):
         newlines.append((next_line_nr, new_next_line))
     return errors, newlines
 
+flexible_tactics = ["rfl", "ring", "aesop", "norm_num", "positivity", "abel", "omega"]
```

Comment:
If the content of this list was determined on zulip, can you please add a link to that discussion in a comment?

Personally, I think `rfl` is not very flexible. Otoh, `linarith` and `nlinarith` probably are?

Reply:
I agree with Johan

Reply:
Seems `linarith` and `nlinarith` were not included in the poll, I'll just add them now.